pkg/restore/restore.go

```diff
@@ -1468,3 +1471,140 @@ func isCompleted(obj *unstructured.Unstructured, groupResource schema.GroupResou
 	// Assume any other resource isn't complete and can be restored
 	return false, nil
 }
+
+type restoreableResource struct {
+	resource                 string
+	selectedItemsByNamespace map[string][]restoreableItem
+	totalItems               int
+}
+
+type restoreableItem struct {
+	path            string
+	targetNamespace string
+	name            string
+}
+
+func (ctx *restoreContext) getOrderedResourceCollection(backupResources map[string]*archive.ResourceItems) (restoreResourceCollection []restoreableResource, warnings Result, errs Result) {
```

Comment:
Our current code style is to not use named return variables. Could you move them inline with the function please?